soql-analyzer/src/main/scala/com/socrata/soql/AnalysisDeserializer.scala

```diff
@@ -204,6 +208,7 @@ class AnalysisDeserializer[C, T](columnDeserializer: String => C, typeDeserializ
       val search = readSearch()
       SoQLAnalysis(
         isGrouped,
+        distinct,
```

Comment:
This should bump the version and add a case to `apply` down below (so that deployment is "release pg secondaries first, then query coordinator" without needing a downtime).